docs/quickstart.md

```diff
@@ -36,7 +36,7 @@ docsify serve docs
 
 ## Manual initialization
 
-If you don't like `npm` or have trouble installing the tool, you can manually create `index.html`:
+If you don't like `npm` or have trouble installing the tool, you can manually create `index.html` (the code below will load the latest version of docsify automatically. If you want a specific version to avoid any broken elements in the future, [see below](#specifying-docsify-versions)):
```

Comment:
You can revert this change.

We do not want people to load the latest version of docsify automatically. We want them to get the latest minor and patch releases, but have to manually update major releases. This will prevent their site from breaking when a new major version is released.